src/wallet/load.cpp

```diff
@@ -109,10 +109,21 @@ void StopWallets()
 void UnloadWallets()
 {
     auto wallets = GetWallets();
+    std::string loaded_wallets;
     while (!wallets.empty()) {
         auto wallet = wallets.back();
+
+        // Write the wallet name to rw conf
+        if (!loaded_wallets.empty()) {
+            loaded_wallets += ",";
+        }
+        loaded_wallets += wallet->GetName();
+
         wallets.pop_back();
         RemoveWallet(wallet);
         UnloadWallet(std::move(wallet));
     }
+    if (gArgs.HaveRWConfigFile()) {
+        gArgs.ModifyRWConfigFile("loadedwallet", loaded_wallets);
```

Comment:
Should just use `wallet=`?

Reply:
That will cause some problems as is since it would be one string that represents multiple wallets. If/when rwconf allows repeated fields, then it can be `wallet=`.